dashboard/test/ui/features/teacher_tools/section_action_dropdown.feature

```diff
@@ -32,10 +32,10 @@ Feature: Using the SectionActionDropdown
     And I wait until current URL contains "/login_info"
 
   @skip
```

Comment:
I'm curious about what is still not working from this one. If you have any notes, could you add them to the description?

Reply:
I actually think this will work.  I have a few more tests that should be easy to add in a later PR.   This PR is more proof that we can get a passing build.  I think I might be able to put together another PR Monday morning with the rest of the tests.